### PR TITLE
uniutils: init @ 2.27

### PR DESCRIPTION
**Pull request by Synthetica9**
Present in 9 repo's at the time of writing:
https://repology.org/project/uniutils/versions

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Review by calbrecht — CHANGES_REQUESTED** [review]
Care to add `outputs = [ "out" "man" ]`?

;) unifuzz is weird.

```
[nix-shell:~/.cache/nixpkgs-review/pr-85540-1]$ echo üöïßœáðïúó | /nix/store/j2np9939p4jv3ay506p484hg1460br35-uniutils-2.27/bin/utf8lookup
/nix/store/j2np9939p4jv3ay506p484hg1460br35-uniutils-2.27/bin/utf8lookup: line 12: ascii2binary: command not found
```

Thanks for this packaging, interesting programs :)

**Comment by calbrecht** [comment]
`nixpkgs-review pr 85540`
1 package built:
uniutils

File: pkgs/tools/text/ascii2binary/default.nix [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    # url = "https://billposer.org/Software/Downloads/${pname}-${version}.tar.bz2";
+    # Use Debian's mirror instead:
+    url = "mirror://debian/pool/main/a/${pname}/${pname}_${version}.orig.tar.gz";
+    sha256 = "1a9md30a60mrr6jri172m88ni5ygj1i8ghdzfgjksl6w5cmk7p5d";
+  };
+
+  meta = with stdenv.lib; {
+    description = "Convert between ASCII, hexadecimal and binary representations";
+    homepage = "https://www.billposer.org/software.html";
+    maintainers = with maintainers; [ synthetica ];
```
Comment (SuperSandro2000) [inline pkgs/tools/text/ascii2binary/default.nix:18]: Please add license and platforms.

File: pkgs/tools/text/uniutils/default.nix [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    ./configure --prefix=$out
+  '';
+
+  buildPhase = ''
+    make
+  '';
+
+  postFixup = ''
+    wrapProgram "$out/bin/utf8lookup" \
+      --prefix PATH ":" ${ascii2binary}/bin:$out/bin
```
Comment (SuperSandro2000) [inline pkgs/tools/text/uniutils/default.nix:48]:
```suggestion
      --prefix PATH ":" "${ascii2binary}/bin"
```

File: pkgs/tools/text/uniutils/default.nix [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+  testPhase = ''
+    cd $src
+    make test
+  '';
+
+  outputs = [ "out" "man" ];
+
+  meta = with stdenv.lib; {
+    description = "A set of programs for manipulating and analyzing Unicode text";
+    homepage = "https://www.billposer.org/software.html";
```
Comment (SuperSandro2000) [inline pkgs/tools/text/uniutils/default.nix:60]: Please add license and platforms.

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]
Please change the commit message to at instead of @.

**Comment by SuperSandro2000** [comment]
Result of `nixpkgs-review pr 85540` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages built:</summary>
  <ul>
    <li>ascii2binary</li>
    <li>uniutils</li>
  </ul>
</details>

**Comment by SuperSandro2000** [comment]
Closing due to inactivity from author.

**Pull request closed without merging**